cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -1984,7 +1984,9 @@ fn emit_return_call_common_sequence<T>(
                  but the current implementation relies on them being present"
     );
 
-    let tmp = call_info.tmp.to_writable_reg();
+    // Hard-coded register which doesn't conflict with function arguments or
+    // callee-saved registers.
+    let tmp = Writable::from_reg(regs::r11());
```

Comment:
This might be true for our current calling conventions, but is not guaranteed to continue to hold true in the future. As such, could you assert that `call_info.uses` doesn't contain any uses of `r11` to future proof this code?